Readme notes

deivid-rodriguez opened the pull request:
Follow up to https://github.com/teampoltergeist/poltergeist/issues/866#issuecomment-299038062.

Not sure about the wording, whether to include links to relevant issues or a more technical explanation about the problem... Just let me know and I'll update it.

I also took the chance to bump the linux links to 2.1.1.

twalpole commented:
Please add a link to the issue #866 after the warning.  Also end the commit description with [ci skip] so it doesn't trigger builds.  Thanks

deivid-rodriguez commented:
* Added a link to the issue.
* Documented the [ci skip] thing.
* Squashed everything to one commit.
* Added [ci skip] to the end of the commit description.

Hope it's fine!

twalpole commented:
Thanks!
